sdk_container/src/third_party/prefix-overlay/skel/etc/portage/package.mask/coreos

```diff
@@ -0,0 +1,2 @@
+app-misc/ca-certificates::coreos # coreos ca-certificates doesn't support prefix
+sys-apps/baselayout::coreos # coreos baselayout depends on systemd
```

Comment:
The packages are fundamentally different to their upstream counterparts, which is a shame. I feel like they don't have to differ quite so much. The coreos baselayout could possibly even be an additional package instead.

Reply:
It depends on the feasibility - but it's sure worth researching. Baselayout in coreos-overlay in particular is very Flatcar specific; if we can manage to use stock Gentoo baselayout then this can be removed here.

Reply:
Using Gentoo baselayout instead of our own stuff would be nice. Maybe it's even possible, who knows. Just no one had a look at it yet. We have a bunch of our own stuff in our baselayout fork, though.

Reply:
But until then, I suppose we would need to do some work on our fork to support eprefix in order to make the prefix overlay come out of experimental status.